Sources/Emoji.swift

```diff
@@ -1431,4 +1431,64 @@ public let emojiList: [String: String] = [
   "yemen": "🇾🇪",
   "zambia": "🇿🇲",
   "zimbabwe": "🇿🇼",
-]
\ No newline at end of file
+  "starstruck": "🤩",
```

Comment:
@Francescu This is cool. However I would like to move these manual added emojies into a new file called ManualAddedEmojis. As this `Emoji.swift` file is generated by a script